transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__performance_events.sql

```diff
@@ -0,0 +1,73 @@
+{{
+    config({
+        "tags":"hourly",
+    })
+}}
+
+WITH MM_TELEMETRY_PROD AS 
+(SELECT _dbt_source_relation	  AS _dbt_source_relation	
+ , context_page_referrer          AS context_page_referrer
+ , channel                        AS channel
+ , context_app_namespace          AS context_app_namespace
+ , id                             AS id
+ , user_actual_id                 AS user_actual_id
+ , context_library_name           AS context_library_name
+ , type                           AS type
+ , context_app_version            AS context_app_version
+ , user_actual_role               AS user_actual_role
+ , context_page_url               AS context_page_url
+ , context_os_name                AS context_os_name
+ , context_page_title             AS context_page_title
+ , anonymous_id                   AS anonymous_id
+ , context_app_build              AS context_app_build
+ , context_page_search            AS context_page_search
+ , context_library_version        AS context_library_version
+ , context_ip                     AS context_ip
+ , context_useragent              AS context_useragent
+ , context_app_name               AS context_app_name
+ , context_locale                 AS context_locale
+ , context_screen_density         AS context_screen_density
+ , context_page_path              AS context_page_path
+ , context_os_version             AS context_os_version
+ , category                       AS category
+ , duration                       AS duration
+ , num_of_request                 AS num_of_request
+ , max_api_resource_size          AS max_api_resource_size
+ , longest_api_resource_duration  AS longest_api_resource_duration
+ , user_id                        AS user_id
```

Comment:
Is there a place in the new codebase to document some of these columns? Like the difference between `user_id` and `userid`? cc: @ifoukarakis , @sri-byte

Reply:
They're both the same, I suspect it was a bug during development that was later resolved. 0/5 on a place to document this.

Reply:
We can coalesce them if that's the case. After all, the staging layer is the perfect place for performing cleanup on source data.

Reply:
Same as above, once I review the data with Harrison, we'll revisit this and make improvements.